### ISSUE TITLE
[1.16.5] CCL rendering error that occasionally causes crashes

### ISSUE THREAD

**Issue by norm1000100** [BluSunrize/ImmersiveEngineering#4912]
I want to be clear and state that I have tested this with and without Optifine installed. As well as on a separate test server and the crash occurs OCCASIONALLY, but usually when loading chunks quickly.

### Description of the issue:
CCL caught an exception rendering a block. Multiple multiblock structures disintegrate into abstract art and pipes disconnect. Sometimes this bug crashes the client, sometimes not. 
![2021-08-09_00 37 46](https://user-images.githubusercontent.com/3014634/128673253-d438f7ea-9f7c-4466-ba16-a146cfb9802f.png)
![2021-08-09_00 37 17](https://user-images.githubusercontent.com/3014634/128673258-d267621e-818c-4f36-87bc-65c932fc817c.png)
![2021-08-09_00 37 23](https://user-images.githubusercontent.com/3014634/128673261-a234cc2b-5536-4531-8757-1420f564d612.png)
![2021-08-09_00 37 29](https://user-images.githubusercontent.com/3014634/128673263-41f5fcc5-d4f3-44aa-920a-5fc6dc72776f.png)
![CCL Caught](https://user-images.githubusercontent.com/3014634/128673421-1c9b9678-b049-42ae-99c5-20a89722a4b2.png)

### Crashlog:
https://pastebin.com/8EcDWT3j

### Versions & Modlist

Forge Version : 36.2.0
IE Version : 1.16.5-5.0.2-137

Modlist:

https://pastebin.com/KVKcnxnX

**Comment by BluSunrize** [comment]
yeah but...your crashlog still has optifine.

Can you produce this issue without it, and then get us your `latest.log` file? That might help more

**Comment by norm1000100** [comment]
I'm out of town and away from my computer for the next week. I'll close this and reopen it when I get back without optifine. As a side note, is there a solution to the crazy multi-structures as even without optifine they are still occurring?

**Comment by BluSunrize** [comment]
No, like I said, that's not an issue I have seen pop up *without* optifine. Which is why I need a latest.log of those glitches happening when optifine is not installed :V

**Comment by norm1000100** [comment]
Hey! I didnt leave as early as I thought so I booted it up and viola! no optifine and the crash occurred.

I'll attach the Paste.ee and latest.log below : 

https://paste.ee/p/ZTlUT

[latest.log](https://github.com/BluSunrize/ImmersiveEngineering/files/6958305/latest.log)


I hope this gives a better picture of whats going on.

Best of luck!
